src/App.js

```diff
@@ -1,25 +1,69 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import axios from "axios";
+import { BrowserRouter as Router, Routes, Route, NavLink } from 'react-router-dom';
+import HomePage from './components/HomePage';
+import ListBeers from './components/ListBeers';
+import NewBeer from './components/NewBeer';
+import RandomBeer from './components/RandomBeers';
+import SingleBeers from './components/SingleBeers';
+import { useEffect ,useState } from "react";
+
 
 function App() {
+
+  const [appArray, setAppArray] = useState();
+  const [randomArray, setRandomArray] = useState(null);
```

Comment:
For those states, you could also fetch that data directly on the components that use them